modules/ols-0-2-0-release-notes.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-2-0-release-notes_{context}"]
+= {ols-long} version 0.2.0
+
+{ols-offical} 0.2.0 is now available on {ocp-product-title} 4.15 or later.
+
+[id="ols-0-2-2-enhancements_{context}"]
+== Enhancements
+
+The following enhancements have been made with {ols-offical} 0.2.0:
+
+* This release introduces the {ols-offical} Operator as a technology preview release. Previously, the {ols-short} Operator was a developer preview release. 
```

Comment:
If you look at the release notes for plain OCP docs, I think terms like tech preview and dev preview are capitalized:
```suggestion
* This release introduces the {ols-offical} Operator as a Technology Preview release. Previously, the {ols-short} Operator was a Developer Preview release. 
```

Reply:
done